training/deepspeech_training/train.py

```diff
@@ -640,8 +640,8 @@ def __call__(self, progress, data, **kwargs):
                         break
 
                     # Reduce learning rate on plateau
-                    if (FLAGS.reduce_lr_on_plateau and
-                            epochs_without_improvement % FLAGS.plateau_epochs == 0 and epochs_without_improvement > 0):
+                    if (FLAGS.reduce_lr_on_plateau and epochs_without_improvement > 0
```

Comment:
let's improve a bit more and put each condition on a new line, ending in `and` ?